Modules/Tests/YosemiteTests/Stores/CardPresentPaymentStoreTests.swift

```diff
@@ -297,6 +297,84 @@ final class CardPresentPaymentStoreTests: XCTestCase {
         XCTAssert(storageAccount?.status == "complete")
     }
 
+    /// Verifies that loadAccounts succeeds when only WCPay succeeds and Stripe fails.
+    ///
+    func test_loadAccounts_succeeds_when_only_wcpay_succeeds() throws {
+        let expectation = self.expectation(description: "Load Account succeeds with WCPay only")
+        network.simulateResponse(requestUrlSuffix: "payments/accounts",
+                                 filename: "wcpay-account-complete")
+        network.simulateResponse(requestUrlSuffix: "wc_stripe/account/summary",
+                                 filename: "generic_error")
+
+        let action = CardPresentPaymentAction.loadAccounts(siteID: sampleSiteID, onCompletion: { result in
+            XCTAssertTrue(result.isSuccess)
+            expectation.fulfill()
+        })
+
+        cardPresentStore.onAction(action)
+        wait(for: [expectation], timeout: Constants.expectationTimeout)
```

Comment:
nit: with `TestKit` import, could simplify the async testing. Similarly for the other 2 test cases.

```suggestion
        // Given
        network.simulateResponse(requestUrlSuffix: "payments/accounts",
                                 filename: "wcpay-account-complete")
        network.simulateResponse(requestUrlSuffix: "wc_stripe/account/summary",
                                 filename: "generic_error")

        // When
        let result = waitFor { promise in
            self.cardPresentStore.onAction(CardPresentPaymentAction.loadAccounts(siteID: self.sampleSiteID, onCompletion: { result in
                XCTAssertTrue(result.isSuccess)
                promise(result)
            }))
        }
        
        // Then
        XCTAssertTrue(result.isSuccess)
```